src/Standards/PEAR/Sniffs/Commenting/ClassCommentSniff.php

```diff
@@ -45,9 +45,7 @@ public function process(File $phpcsFile, $stackPtr)
     {
         $this->currentFile = $phpcsFile;
 
-        $tokens    = $phpcsFile->getTokens();
-        $type      = strtolower($tokens[$stackPtr]['content']);
```

Comment:
:question:  Looks like `$type` and `$errorData` should be used (in the error message), but the usage got accidentally removed when the new comment tokenizer got implemented. Relevant commits: 371ec577d810899b6e1122f7e8118405b6a7de53, 5292c84edf3d1c00610df34362397825a159ed3a and 22060b65a3cb54a49011d87fd90cb3dac5388746